Cargo.toml

```diff
@@ -16,6 +16,7 @@ exclude = ["doc-template", "plotters-doc-data"]
 num-traits = "0.2.14"
 chrono = { version = "0.4.19", optional = true }
 plotters-svg = {version = "^0.3.*", optional = true}
+base64 = "*"
```

Comment:
I am a little bit hesitate on this, two things:
first, we should be conservative on introducing new unconditional dependencies. In fact, building time for plotters with minimal feature is one of our major goal. As I discussed with downstream crates (especially criterion), they don't want to introduce additional dependency to build.
Second, you should use a version, instead of star - otherwise, cargo won't allow publish your crate.

Actually, I have a very minimal version base64 implementation available with the SVG backend. So I think it would be nice if you can reuse that. 

I mean create a SVGWrapper and then call blit_bitmap